manifests/addons/dashboards/istio-mesh-dashboard.json

```diff
@@ -1461,7 +1461,7 @@
           "link": false,
           "linkTargetBlank": false,
           "linkTooltip": "$__cell dashboard",
-          "linkUrl": "/dashboard/db/istio-workload-dashboard?var-namespace=${__cell_3:raw}&var-workload=${__cell_2:raw}",
+          "linkUrl": "/d/12010103e01030/istio-workload-dashboard?var-namespace=${__cell_3:raw}&var-workload=${__cell_2:raw}",
```

Comment:
this seems not correct? should be `UbsSZTDik` ? from https://github.com/istio/istio/blob/f611f67505bb60c469f93aca9de0d7f14e8d3538/manifests/addons/dashboards/istio-workload-dashboard.json#L2670

Reply:
you're right, I had an older version of the dashboard with this uid set, this has been fixed.